package/tvheadend-app-osmc/files/DEBIAN/postinst

```diff
@@ -41,6 +94,61 @@ if [ "$1" = "configure" ]; then
 		fi
 	fi
 
-	if ischroot; then exit 0; fi
+	# start tvh so that it updates any old user files
+	echo Starting tvheadend to finalise setup for osmc
 	if systemctl is-enabled tvheadend.service >/dev/null; then systemctl start tvheadend.service; fi
+	#systemctl restart tvheadend
+	sleep 5
```

Comment:
I've not tried it without the sleep, maybe not needed.  TVH has migration code in it which I think must read the previous version from config.  That's why my first "upgrade" attempt failed because my user files were Version 4.1 (upgraded properly) and the one generated by postinst was 4.0 which got mangled. The alternative is to hack the user files before tvh starts, but we couldn't add the two parameters into the 4.0.9 files as it doesn't support them.